packages/create-next-stack/src/main/plugins/panda-css/panda-css.ts

```diff
@@ -0,0 +1,121 @@
+import endent from "endent"
+import { makeDirectory, writeFile } from "../../helpers/io"
+import { createPlugin } from "../../plugin"
+import { generateIndexPage } from "./add-content/pages/generate-index"
+
+export const pandaCSSPlugin = createPlugin({
+  id: "panda-css",
+  name: "Panda CSS",
+  description: "Adds support for Panda CSS",
+  active: ({ flags }) => flags["styling"] === "panda-css",
+  devDependencies: {
+    pandacss: {
+      name: "@pandacss/dev",
+      version: "^0.x.x",
+    },
+    autoprefixer: {
+      name: "autoprefixer",
+      version: "^10.0.0",
+    },
+    postcss: {
+      name: "postcss",
+      version: "^8.0.0",
+    },
+  },
+  technologies: [
+    {
+      id: "pandaCSS",
+      name: "Panda CSS",
+      description:
+        "🐼 Universal, Type-Safe, CSS-in-JS Framework for Product Teams ⚡️",
+      links: [
+        { title: "Website", url: "https://panda-css.com" },
+        { title: "Docs", url: "https://panda-css.com/docs" },
+        { title: "GitHub", url: "https://github.com/chakra-ui/panda" },
+      ],
+    },
+  ],
+  steps: {
+    setUpPandaCss: {
+      id: "setUpPandaCss",
+      description: "setting up panda CSS",
+      run: async () => {
+        await Promise.all([
+          addPandaConfig(),
+          addPostcssConfig(),
+          addStylesGlobalsCss(),
+        ])
+      },
+    },
+    addContent: {
+      id: "addContent",
+      description: "adding content",
+      run: async (inputs) => {
+        await Promise.all([
+          writeFile("pages/index.tsx", generateIndexPage(inputs)),
```

Comment:
@akd-io 
As it seems this part will not overwrite the original `pages/index.tsx`. I guess this part is executed before `createNextStackPlugin`. 
But here it noted that steps run form top to bottom https://github.com/akd-io/create-next-stack/blob/b0aa3ae4afd152079dfb53d36f16f483474a6813/CONTRIBUTING.md?plain=1#L99